This patch is needed because our chart-iamguarded-postgresql is at 17.x while upstream references 16.x.x. This is the same pattern used in keycloak chart.

diff --git a/bitnami/kong/Chart.yaml b/bitnami/kong/Chart.yaml
index 6213aa99ee..f5dff21d1e 100644
--- a/bitnami/kong/Chart.yaml
+++ b/bitnami/kong/Chart.yaml
@@ -13,7 +13,7 @@ dependencies:
 - condition: postgresql.enabled
   name: postgresql
   repository: oci://registry-1.docker.io/bitnamicharts
-  version: 16.x.x
+  version: 17.x.x
 - name: common
   repository: oci://registry-1.docker.io/bitnamicharts
   tags:
